app/serializers/statistic_info_chart_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class StatisticInfoChartSerializer
   include FastJsonapi::ObjectSerializer
 
-  attributes :hash_rate, :difficulty
+  attributes :hash_rate, :difficulty, uncle_rate
```

Comment:
It will be better to replace with `hash_rates` , `difficulties` and `uncle_rates`

Reply:
I don't think so, I treat it as a piece of data in the statistic info。